app/controllers/dashboard_controller.rb

```diff
@@ -0,0 +1,5 @@
+class DashboardController < ApplicationController
+   def index
+    @github_user = GithubUser.find_by(current_user.oauth_token)
```

Comment:
this line returns:

"An HTTP request has been made that VCR does not know how to handle:
  GET https://api.github.com/user?access_token=

VCR is currently using the following cassette:
  - /Users/sergeylukyanenko/turing/module3/apicurious/spec/fixtures/vcr_cassettes/github_user_logs_in.yml
    - :record => :once
    - :match_requests_on => [:method, :uri]..."
@case-eee